files/zh-cn/web/javascript/reference/operators/delete/index.html

```diff
@@ -51,7 +51,7 @@ <h3 id="异常">异常</h3>
 
 <h2 id="Description" name="Description">描述</h2>
 
-<p>与通常的看法不同，<code>delete</code>操作符与直接释放内存<strong>无关</strong>。内存管理 通过断开引用来间接完成的，查看<a href="https://developer.mozilla.org/zh-CNdocs/Web/JavaScript/Memory_Management">内存管理</a>页可了解详情。</p>
+<p>与通常的看法不同，<code>delete</code>操作符与直接释放内存<strong>无关</strong>。内存管理 通过断开引用来间接完成的，查看<a href="https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Memory_Management">内存管理</a>页可了解详情。</p>
```

Comment:
Please remove the host, internal page should use relative URL.

Reply:
Fixed!